lib/cocoapods/generate/installer.rb

```diff
@@ -109,7 +109,11 @@ def create_app_project
         spec.available_platforms.map do |platform|
           consumer = spec.consumer(platform)
           target_name = "App-#{Platform.string_name(consumer.platform_name)}"
-          Pod::Generator::AppTargetHelper.add_app_target(app_project, consumer.platform_name, deployment_target(consumer), target_name)
+          native_app_target = Pod::Generator::AppTargetHelper.add_app_target(app_project, consumer.platform_name, deployment_target(consumer), target_name)
+          # Temporarily set Swift version to pass validator checks for pods which do not specify Swift version.
+          # It will then be re-set again within #perform_post_install_steps.
+          Pod::Generator::AppTargetHelper.add_swift_version(native_app_target, '3.2')
```

Comment:
please extract this out into a constant

Reply:
should I reuse the same one from `validator.rb` ?

Reply:
https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/validator.rb#L19

Reply:
👍

Reply:
done